### PR TITLE
DOC-2270: Add focus style changes to Accompanying Enhanced Skins & Icon Packs changes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2294

Site: [DOC-2270_DOC-2294 site](http://docs-feature-70-doc-2270doc-2294.staging.tiny.cloud/docs/tinymce/latest/7.0-release-notes/#accompanying-enhanced-skins-and-icon-packs-changes)

Changes:
* Add focus style changes to Accompanying Enhanced Skins & Icon Packs changes.

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 1]
```diff
@@ -100,13 +100,24 @@ The following Premium plugin has been announced as reaching its end-of-life:
...
+The {productname} 7.0 release includes an accompanying release of the **Enhanced Skins & Icon Packs**.
 
 === Enhanced Skins & Icon Packs
 
 The **Enhanced Skins & Icon Packs** release includes the following updates:
 
-The **Enhanced Skins & Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} <x.y[.z]> skin, Oxide.
+The **Enhanced Skins & Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 7.0 skin, Oxide.
+
+==== Fixed some focus issues related to {productname} 5 skin and premium skins
```
Comment (ltrouton) [inline modules/ROOT/pages/7.0-release-notes.adoc:111]: Do you reckon mentioning TinyMCE 5 in the heading could add potential confusion? Could we just omit `5` from the heading?

**Review by ltrouton — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.0-release-notes.adoc:111]:
```suggestion
==== Fixed focus issues related to Enhanced Skins and Icon Packs
```

Comment (MitchC1999) [reply modules/ROOT/pages/7.0-release-notes.adoc:111]: Isn't this change relevant to all skins though?

Comment (kemister85) [reply modules/ROOT/pages/7.0-release-notes.adoc:111]: Yeah, 😢

Comment (kemister85) [reply modules/ROOT/pages/7.0-release-notes.adoc:111]: I've updated the suggestion, unless anyone else has a better way of explaining it?

**Review by spocke — APPROVED** [review]

**Pull request merged**